src/components/List_Products.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { saveItem } from '../lib/api';
+import { db } from '../lib/firebase';
+import { collection, onSnapshot } from 'firebase/firestore';
+
+function ListProducts() {
+  const [items, setItems] = useState(null);
+  const [itemName, setItemName] = useState('');
+
+  /* Get items */
+  useEffect(() => {
+    const unsubscribe = onSnapshot(collection(db, 'Products'), (snapshot) => {
+      let products = [];
+
+      snapshot.forEach((doc) => {
+        products.push({ ...doc.data(), id: doc.id });
```

Comment:
why not do `setItems({ ...doc.data(), id: doc.id })` directly instead of using an extra `products` array?